packages/embeds/embed-core/src/embed.ts

```diff
@@ -52,6 +52,7 @@ if (!globalCal || !globalCal.q) {
 // TODO: Ideally it should be the version as per package.json and then it can be renamed to version.
 // But because it is built on local machine right now, it is much more reliable to have the commit hash.
 globalCal.fingerprint = process.env.EMBED_PUBLIC_EMBED_FINGER_PRINT as string;
```

Comment:
Still keeping the fingerprint as that confirms the commit used to build it. Version could still go wrong but fingerprint can't be wrong.